### ISSUE TITLE
Tasker support in v5

### ISSUE THREAD

**Issue by szaimen** [blokadaorg/blokada#816]
Hey, would be great if v5 could bring back Tasker support. Thanks!

**Comment by pfactum** [comment]
Yes please! If this gets implemented, then I guess #808 gets resolved automagically.

**Comment by andreapx** [comment]
Just to know: is this feature been developed or not yet? Is there any ETA?
Thanks!!!

**Comment by peterroth** [comment]
@andreapx I was working on it, but couldn't finish due to changes in my life.
